packages/auth-plugins/src/UserUtils.ts

```diff
@@ -28,24 +31,24 @@ export function getUserFromConfig(
   function getValue(key: string): string | undefined {
     return getAppInitValue(serverConfig, key);
   }
-  function getBooleanValue(key: string, defaultValue = true): boolean {
-    if (defaultValue) {
-      return !(getValue(key) === 'false');
+  function getBooleanValue(key: string, defaultValue: boolean): boolean {
+    const value = getValue(key);
+    if (value === 'true') {
+      return true;
+    }
+    if (value === 'false') {
+      return false;
+    }
+    if (value !== undefined) {
+      log.warn(`Unexpected value for ${key}: ${value}`);
     }
-    return getValue(key) === 'true';
+    return defaultValue;
   }
   const name = getValue('name') ?? '';
   const operateAs = getValue('operateAs') ?? name;
   const groups = getValue('groups')?.split(',') ?? [];
```

Comment:
Are these part of DHC? Or will they be? I'm not sure if these are used in DnD

Reply:
Not used in DHC. I didn't want to make any breaking changes as part of this change.
The `User` interface is used in Enterprise.

Reply:
https://github.com/deephaven/web-client-ui/issues/1401